plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -74,15 +89,16 @@
   private IProject project;
   private Credential credential;
   protected IPath workDirectoryParent;
-  private ProcessOutputLineListener stdoutLineListener;
+  private MessageConsoleWriterOutputLineListener stdoutLineListener;
   private ProcessOutputLineListener stderrLineListener;
   private DefaultDeployConfiguration deployConfiguration;
   private CollectingLineListener errorCollectingLineListener;
+  private int stagingOutputEndIndex = 0;
```

Comment:
Perhaps the exit-listener should just copy out the content on success?
`String rawDeployOutput`?